recipes/essentials/recipe.toml

```diff
@@ -0,0 +1,10 @@
+description = "thin-edge.io essential recipes"
+dependencies = [
+    "thin-edge.io",
+    "persist-data",
+    "persist-overlay",
+    "tedge-bootstrap",
+    "tedge-firmware-update",
+    "avahi-daemon",
+    "mosquitto",
```

Comment:
Mosquitto is listed here and in the defaults recipe. Is this an issue?

Reply:
No issue but an oversight. Good catch! I will fix that.